hicommonwealth/commonwealth
# Block /profiles in robots.txt and then change crawl priority.

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8690

## Description of Changes
- block /profile
- set crawl priority properly

## "How We Fixed It"


## Test Plan
- write a new sitemap
- load the sitemap
- verify there are no profile URLs
- verify thread priority
- verify robots.txt has the disallow rule for /profile

## Deployment Plan

1. 

## Other Considerations

-
